source/Extractors/src/ExtractorFCCSW.cc

```diff
@@ -158,8 +158,8 @@ bool ExtractorFCCSW::analyze()
   xmlSubdetectorBeampipe->SetAttribute("name", "beampipe");
   xmlSubdetectorBeampipe->SetAttribute("type", "BeamTube");
   auto xmlBeamPipeDimensions = m_xmlDoc->NewElement("dimensions");
-  xmlBeamPipeDimensions->SetAttribute("rmin", printWithUnit(0., c_precision, "mm").c_str());
-  xmlBeamPipeDimensions->SetAttribute("rmax", printWithUnit(15., c_precision, "mm").c_str());
+  xmlBeamPipeDimensions->SetAttribute("rmin", printWithUnit(20., c_precision, "mm").c_str());
+  xmlBeamPipeDimensions->SetAttribute("rmax", printWithUnit(21., c_precision, "mm").c_str());
   xmlBeamPipeDimensions->SetAttribute("z", printWithUnit(500., c_precision, "mm").c_str());
   xmlBeamPipeDimensions->SetAttribute("material", "Air");
```

Comment:
The problem was actually not that big -- the beampipe we are using is in DetCommon, on which a lot of other subdirectories "depend" as you say in Gaudi. If I link ACTS in DetCommon (which I need to add the ACTS extensions) all other Gaudi subdirectories will start to print CMake warnings. We haven't figured out why yet, or how to stop this. I suspect that that the issue will disappear with some recent changes in ACTS, that's why I proposed to leave the Beampipe hear temporarily. We need the new LCG release to use more recent versions of ACTS). I agree that it is not ideal ...